src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1225,6 +1225,18 @@ window.qBittorrent.DynamicTable = (function() {
                     if (~state.indexOf('paused'))
                         return false;
                     break;
+                case 'stalled':
+                    if(state != 'stalledUP' && state != 'stalledDL')
```

Comment:
Add space: `if (...` and add parentheses for each not-equal comparison.

Reply:
@Chocobo1 done.